apps/aecore/src/aec_metrics_rpt_dest.erl

```diff
@@ -170,11 +170,17 @@ make_destinations(Found, Actions0, Tab) ->
     Dests = mk_dests_(Actions0),
     lists:foreach(
       fun({Metrics, DPs}) ->
-              [ets:insert(Tab, #metric{key = {N,D}, destinations = Dests})
+              [ets_insert_new(Tab, #metric{key = {N,D}, destinations = Dests})
                || {N, _, _} <- Metrics,
                   D <- DPs]
       end, Found).
 
+ets_insert(Tab, Obj) ->
```

Comment:
Why is this function introduced?
Perhaps it should have a less specific name to allow for future rewrite to use another backend?

Reply:
It was introduced to simplify debugging via tracing. I usually add a comment to that effect.